### PR TITLE
Update Relatives subsection so that "I Dont Know" clears branches

### LINKED ISSUE #1466: Relatives - Checkboxes/Branching
## Where are you in the form?

**Section:**  Relationships

**Question:** Relatives
 
**Field:**  Multiple, see below

**Component (for example address block):** Accordion > Address Block, and others

## What did you see?

Checkboxes should be clearing data, but they do not leaving multiple selections chosen. See screenshots. 

`Provide the address of current employer, or provide the address of their most recent employer if not currently employed`
If you select "I don't know" the radio buttons should clear for type of addresses.
screen shot 2019-02-21 at 3 20 58 pm

`Is this relative affiliated with a foreign government, military, security, defense industry, foreign movement, or intelligence service?`
If you select "I don't know" the radio buttons should clear for yes or no and delete the followup question and text box. 
screen shot 2019-02-21 at 3 33 09 pm

- [x] Validation error
- [x] Data not saving
- [ ] Language or content issue
- [ ] Other (explain below)

## What did you expect to see?

See intended behavior above.
This feels like an issue that needs more general attention to understand what the Relatives module is doing.

## Is this error limited to just this field or others in the form?

- [ ] Only relevant to just this question
- [ ] Cross-form issue
- [x] I don't know

## What operating system and browser/version are you using?

**Operating system**
- [ ] Windows
- [x] MacOS

**Browser**
- [x] Chrome
- [ ] Firefox
- [ ] IE
- [ ] Edge
- [ ] Other

**Version:**

### PR DESCRIPTION
**Pull request by christopherhuii**
## Description
Fixes #1466 

If you selected `I Don't Know` for the following questions, all data and branching will be removed.

1. `Provide the address of current employer, or provide the address of their most recent employer if not currently employed`

2. `Is this relative affiliated with a foreign government, military, security, defense industry, foreign movement, or intelligence service?`

## Checklist for Reviewer

- [ ] Review code changes
- [ ] Review changes for Database effects
- [ ] Verify change works in IE browser

More details about this can be found in [docs/review.md](docs/review.md)

### CODE REVIEW

File: src/components/Section/Relationships/Relatives/Relative.jsx [hunk 1]
```diff
@@ -19,65 +17,164 @@ import {
...
+      name: 'relation-motherinlaw',
+      label: i18n.m('relationships.relatives.label.relation.motherinlaw'),
+      value: 'Mother-in-law',
+    }, {
+      name: 'relation-guardian',
+      label: i18n.m('relationships.relatives.label.relation.guardian'),
+      value: 'Guardian',
+    },
+  ])
+
```
Comment (suzubara) [inline src/components/Section/Relationships/Relatives/Relative.jsx]: The options defined in these functions seem like they'd be good to store in another file eventually.

**Comment by suzubara** [comment]
Looking at the original issue, it looks like the address type (In the U.S.) should be un-selected if you choose "I don't know", however on this branch it still appears to be selected.

**Comment by christopherhuii** [comment]
I added the ability for the `Address` component to not have an address type. However, this makes things a bit more complicated in how the address fields display. I added some details [here](https://github.com/18F/e-QIP-prototype/issues/1466#issuecomment-479135538).

I also moved the `Is this relative affiliated with a foreign government, military, security, defense industry, foreign movement, or intelligence service?` explanation texture underneath the `I don't know` button as indicated [here](https://github.com/18F/e-QIP-prototype/issues/1466#issuecomment-478638365).

**Comment by ryanhofdotgov** [comment]
@christopherhuii Regarding un-typed Addresses, what changes would the backend see? Is `isEnabled` a new property that's going to start manifesting itself in the serialized JSON?

**Comment by christopherhuii** [comment]
`isEnabled` does not get sent to the backend. The backend does not receive anything new in the JSON.

**Comment by ryanhofdotgov** [comment]
@christopherhuii @suzubara It seems with this change, `I don't know` has a different behavior in this one section from all the other places where there is an address fieldset/`I don't know` button. For example, Foreign Contacts.

Ditto with other fields (like email) where toggling an associated `I don't know` preserves any existing value in that field. 

What's involved in making the behavior consistent across the app?

**Pull request merged**